library/src/morpheus-master/sass/base/_defaults.scss

```diff
@@ -119,21 +119,20 @@ input[type="checkbox"], input[type="radio"]{
 	width: 16px;
 	vertical-align: sub;
 	@include appearance(none);
-	&:checked{
-		border: 1px solid #60b3e3;
-	}
+
 	//SAK-30977
 	&[disabled],&[disabled="disabled"],&[disabled="true"]{
 		opacity: 0.8;
```

Comment:
I think this opacity should be "0.5" to make the difference between an enabled and disabled input element more clear.

I see that Sam mentioned that this should be darkened, but according to WCAG2 (https://www.w3.org/TR/WCAG21/#non-text-contrast), the contrast ratio is 3:1 "except for inactive components".

Reply:
Is it inactive and thus irrelevant to the user with poor visability? Or is it actually an important element that will become relevant once another input is clicked?

Hiding truly inactive inputs is fine. But if this is an important element, why make it so difficult for users to read?

https://axesslab.com/disabled-buttons-suck/

Reply:
That is a good question. 

What would be an example of an important disabled element? When is a disabled element important?

Reply:
I think opacity:0.6 will give us enough minimum contrast to meet text contrast requirements and (hopefully) be distinct enough from enabled elements.